fe/fe-core/src/main/java/com/starrocks/metric/ResourceGroupMetricMgr.java

```diff
@@ -0,0 +1,232 @@
+// This file is made available under Elastic License 2.0.
+// This file is based on code available under the Apache license here:
```

Comment:
Is this file copied from Apache Doris?

Reply:
Not from the doris project, but other classes from starrocks. I will replace it with "// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited."

Reply:
If not from the apache doris, the`Apache license` line is not necessary. StarRocks project uses Elastic liscense.